charts/spegel/templates/servicemonitor.yaml

```diff
@@ -6,10 +6,15 @@ metadata:
   namespace: {{ include "spegel.namespace" . }}
   labels:
     {{- include "spegel.labels" . | nindent 4 }}
+    {{- if .Values.serviceMonitor.labels -}}
```

Comment:
Use with instead of if here.

Reply:
`with` is used with [predefined key](https://helm.sh/docs/chart_template_guide/control_structures/#modifying-scope-using-with) but in this case key:value of serviceMonitor specific labels are arbitrary
`range` can be used like this to produce same result

```
  labels:
    {{- include "spegel.labels" . | nindent 4 }}
    {{- range $k, $v := .Values.serviceMonitor.labels }}
    {{ $k }}: {{ $v }}
    {{- end }}
```

let me know if you meant something different for using `with` instead of `if` or `range`

Reply:
In that case I think keeping the if is better.

Reply:
made the changes in #220 PR